pkg/app/run_test.go

```diff
@@ -303,6 +303,10 @@ COMMANDS
         --version=VERSION        Number of version to activate
     -p, --path=PATH              Path to package
 
+  compute publish
+    Runs "build" then "deploy" using fastly.toml manifest configuration
+
+
```

Comment:
@fgsch @phamann this extra line break is annoying but it looks to be a consequence of not having any flags (TBD! see convo so far in [issue here](https://github.com/fastly/cli/issues/232#issuecomment-810946616)).

If we actually add flags, then this extra line break will go away.